nucypher/blockchain/eth/policies.py

```diff
@@ -38,23 +39,23 @@ class BlockchainArrangement(Arrangement):
     federated = False
 
     def __init__(self,
-                 author: PolicyAuthor,
-                 miner: Miner,
+                 alice: PolicyAuthor,
+                 ursula: Miner,
                  value: int,
                  expiration: maya.MayaDT,
                  *args, **kwargs) -> None:
 
-        super().__init__(alice=author, ursula=miner, *args, **kwargs)
+        super().__init__(alice=alice, ursula=ursula, expiration=expiration, *args, **kwargs)
 
         delta = expiration - maya.now()
         hours = (delta.total_seconds() / 60) / 60                               # type: int
-        lock_periods = int(math.ceil(hours / int(constants.HOURS_PER_PERIOD)))  # type: int
+        lock_periods = int(math.ceil(hours / HOURS_PER_PERIOD))  # type: int
 
         # The relationship exists between two addresses
-        self.author = author                     # type: PolicyAuthor
-        self.policy_agent = author.policy_agent  # type: PolicyAgent
+        self.author = alice                     # type: PolicyAuthor
```

Comment:
Should this be `self.alice` or do you intend to keep `author` here?

Reply:
Its kinda where the rubber meets the road - The is a class named `PolicyAuthor` that alice interests from - so which is it?

Reply:
Makes sense. :+1: